tests/test_transpile.py

```diff
@@ -280,3 +280,13 @@ def test_error_level(self, logger):
 
         with self.assertRaises(ParseError):
             transpile("x + 1 (")
+
+    def test_lambda(self):
```

Comment:
can you delete these unit tests, they aren't necessary, or simply add these variants to identity.sql

Reply:
Thanks, I thought that might be the case. I wrote the test before I discovered the extensive list in identity.sql